pom.xml

```diff
@@ -21,6 +21,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-thymeleaf</artifactId>
+        </dependency>
+
```

Comment:
Jeśli w całym POM'ie nie robimy odstępów między poszczególnymi _dependency_ , tutaj też bym odstępów nie robił.

Reply:
To chyba i tak pójdzie do skasowania, thymeleafa używam do testowania. U siebie mam templatki takie jak formularz rejestracji czy logowania, ktore z niego korzystają